src/lib/CellTemplates/NumberCellTemplate.tsx

```diff
@@ -62,13 +64,16 @@ export class NumberCellTemplate implements CellTemplate<NumberCell> {
     }
 
     getClassName(cell: Compatible<NumberCell>, isInEditMode: boolean): string {
-        return cell.className ? cell.className : '';
+        const isValid = cell.validator?.(cell.value) ?? true;
+        const className = cell.className ? cell.className : '';
```

Comment:
I've overlooked that before.
`cell.className ?? ''`

Reply:
Our code again haha, but please throw that in there if possible 🙇‍♂️

Reply:
I did: cell.className || ''